Algorithms/Numeric/TemperatureConverter.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Algorithms.Numeric
+{
+    public static class Converter
+    {
+        static void Main(string[] args)
+        {
+
+            double C;
+            double K;
+            double F;
+
+            Console.WriteLine("If you want to convert from Celsius to Fahrenheit ... type 1 and press enter: ");
+            Console.WriteLine("If you want to convert from Fahrenheit to Celsius ... type 2 and hit enter:");
+            Console.WriteLine("If you want to convert from Fahrenheit to Kelvin ... type 3 and hit enter:");
+            Console.WriteLine("If you want to convert from Kelvin to Fahrenheit ... type 4 and hit enter:");
+            Console.WriteLine("If you want to convert from Kelvin to Celsius ... type 5 and hit enter:");
+            Console.WriteLine("If you want to convert from Celsius to Kelvin ... type 6 and hit enter:");
+            int num = int.Parse(Console.ReadLine());
+                switch (num)
+                {
+                    case 1:
+                        Console.Write("Enter a number in Celsius, which will be converted to Fahrenheit: ");
+                        C = Convert.ToDouble(Console.ReadLine());
+                        F = C * 9 / 5 + 32;
+                        Console.Write(C + "°C is equal to ");
+                        Console.WriteLine(F + " Fahrenheit \n \n");
+                        Console.WriteLine("Formula used: F = C * 9 / 5 + 32");
+                    break;
+
+                    case 2:
+                        Console.Write("Enter a number in Fahrenheit, which will be converted to  Celsius: ");
+                        F = Convert.ToDouble(Console.ReadLine());
+                        C = ((F - 32) * 5 / 9);
+                        Console.Write(F + "°F is equal to ");
+                        Console.WriteLine(C + " Celsius \n \n");
+                        Console.WriteLine("Formula used: C = (F-32) * 5/9");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6301588)